install_config/upgrading/manual_upgrades.adoc

```diff
@@ -363,18 +419,32 @@ renamed service on the node host:
 
 endif::[]
 ifdef::openshift-enterprise[]
-. On the node host, upgrade all *atomic-openshift* packages:
+. Upgrade the node component packages or related images.
+
+.. For nodes using the RPM-based method on a RHEL 7 system, upgrade all
+*atomic-openshift* packages:
 +
 ----
 # yum upgrade atomic-openshift\*
 ----
 
-. Restart the *atomic-openshift-node* service and review its logs to ensure
-services have been restarted successfully:
+.. For nodes using the containerized method on a RHEL 7 or RHEL Atomic Host 7
+system, set the `*IMAGE_VERSION*` parameter in the
+*_/etc/sysconfig/atomic-openshift-node_* and *_/etc/sysconfig/openvswitch_*
+files to the version you are upgrading to. For example:
++
+----
+IMAGE_VERSION=v3.2.1.1
+----
+
+. Restart the *atomic-openshift-node* and *openvswitch* services and review their
+logs to ensure services have been restarted successfully:
```

Comment:
I just found out that IBM says not use possessive case with inanimate objects (p41). So s/their/the